brave/brave-ios
# Fix #5730: Add redirect tracking URLs (debounce) toggle

Move de-amp and debounce to its own section



## Summary of Changes


This pull request fixes #5730 

## Submitter Checklist:


## Test Plan:

1. Enable debouncing
2. Go to https://dev-pages.brave.software/navigation-tracking/debouncing.html
3. Click on Run on default
4. Ensure observed referrer is `https://dev-pages.brave.software/`
5. Disable debouncing
6. Go to https://dev-pages.brave.software/navigation-tracking/debouncing.html
7. Click on Run on default
9. Error page is visible

## Screenshots:

![Simulator Screen Shot - iPhone 14 Pro - 2022-11-11 at 14 51 36](https://user-images.githubusercontent.com/909331/201354124-4ebfa9b2-d4fb-4dbe-ae03-c38aca5bd6b9.png)
![Simulator Screen Shot - iPhone 14 Pro - 2022-11-11 at 14 51 26](https://user-images.githubusercontent.com/909331/201354160-c9024d24-d5cd-4877-b399-3129cbe62422.png)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
